.drone.star

```diff
@@ -621,6 +621,7 @@ basicTestSuites = [
     "webUILogin",
     "webUIMarkdownEditor",
     "webUIMoveFilesFolders",
+    "webUIOperationsWithFolderShares",
```

Comment:
There are more places to add this suite name - search for `webUIMoveFilesFolders` - there are groups of pipelines that run with oC10 and with OCIS, and with different typical phone-sized screen resolutions.